config/archiver.conf

```diff
@@ -0,0 +1,52 @@
+##### Archiver properties #####
+
+# Time sleep in seconds to run the archiving routine (default = 300)
+saps_execution_period_archiver=300
+# Time sleep in seconds to run the failed task cleanup routine (default = 30)
+saps_execution_period_garbage_collector=30
+# Debug mode (default = true)
+# When executing in debug mode, the output generated by failed tasks are also kept in the permanent storage
+saps_debug_mode=true
+
+##### Catalog #####
+
+datastore_url_prefix=jdbc:postgresql://
+datastore_ip=
+datastore_port=
+datastore_name=
+datastore_driver=org.postgresql.Driver
+datastore_username=
+datastore_password=
+
+##### Temporary Storage (NFS) #####
+# Path mounted by the client
+saps_temp_storage_path=
+
+##### Permanent storage (Swift) #####
+saps_permanent_storage_type=swift
+# Folder prefix for to archive failed tasks case debug mode is true (default = trash)
+permanent_storage_debug_tasks_dir=trash
+# Folder prefix for to archive success tasks (default = archiver)
+permanent_storage_tasks_dir=archiver
+# Container name
+swift_container_name=
+# Username
+swift_username=
+# Password
+swift_password=
+# Identity URL + version (e.g. https://<domain>:5000/v1)
+swift_auth_url=
+
+swift_object_store_key=
+
+##### Fogbow Keystone #####
```

Comment:
are we still using fogbow?

Reply:
This Archiver doc is "old", in this case, before changing the constants, so continue with these remnants